rasa/nlu/constants.py

```diff
@@ -39,19 +39,28 @@
 
 NUMBER_OF_SUB_TOKENS = "number_of_sub_tokens"
 
-MESSAGE_ATTRIBUTES = [TEXT, INTENT, RESPONSE]
-DENSE_FEATURIZABLE_ATTRIBUTES = [TEXT, RESPONSE]
+MESSAGE_ATTRIBUTES = [TEXT, INTENT, RESPONSE, MESSAGE_ACTION_NAME, ACTION_TEXT]
```

Comment:
```suggestion
MESSAGE_ATTRIBUTES = [TEXT, INTENT, RESPONSE, ACTION_NAME, ACTION_TEXT]
```

Reply:
Done with changes to `MESSAGE_ACTION_NAME`! I just changed the `message_action_name` everywhere in one go